ElrondNetwork/api.elrond.com
# Add MexAnalytics endpoints

## Type

## Problem setting
- No public endpoints for mex tokens informations 
  
## Proposed Changes
- Add mex analytics endpoints
- Warm mex pairs & mex economics every minute

## How to test
- Check if /mex-economics , /mex-settings, /mex-pairs return same values
- Check if warm runs every minute
